templates/hexagonal-spring-boot-kotlin/jpa-adapter/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>artifactName</artifactId>
+    <groupId>org.dfm</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>jpa-adapter</artifactId>
+  <dependencies>
+    <!-- Domain -->
+    <dependency>
+      <groupId>org.dfm</groupId>
+      <artifactId>domain-api</artifactId>
+    </dependency>
+    <!-- Frameworks & Libraries -->
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-data-jpa</artifactId>
+      <version>${spring-boot.version}</version>
+      <exclusions>
+        <exclusion>
+          <artifactId>byte-buddy</artifactId>
+          <groupId>net.bytebuddy</groupId>
+        </exclusion>
+      </exclusions>
```

Comment:
exclusions are to be managed in dependencyManagement of main pom.xml

Reply:
This exclusion belongs to the spring-boot-starter-data-jpa dependency which we are defining directly in the JPA adapter layer.

Reply:
Manage the dependency like

**pom.xml**
https://github.com/devs-from-matrix/hexagonal-spring-boot-java/blob/master/pom.xml#L81-L91

**jpa-adapter/pom.xml**
https://github.com/devs-from-matrix/hexagonal-spring-boot-java/blob/master/jpa-adapter/pom.xml#L19-L22